unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
@omtinez And I see that there are already LWIP qstr's here:
https://github.com/micropython/micropython/blob/master/py/qstrdefs.h#L618-L645

and settimeout is already defined in there.